dev_tools/bash_scripts_test.py

```diff
@@ -64,6 +64,8 @@ def run(
 dir=$(git rev-parse --show-toplevel)
 cd {}
 git init --quiet
+git config --global user.name 'Me'
```

Comment:
When this is executed locally, it overrides the local git config. Your most recent commits were done by `Me`: https://github.com/quantumlib/Cirq/pull/3038/commits

Reply:
LOL doh.